src/seedsigner/views/view.py

```diff
@@ -243,3 +243,50 @@ def run(self):
         ).display()
         
         return Destination(MainMenuView, clear_history=True)
+
+class MicroSDToastView(View):
+    def event_handler(self, action, mount_dir):
+        import time
+        
+        self.current_screen = self.renderer.canvas.copy()
+        
+        # Special case when screensaver is running
+        if self.controller.screensaver._is_running:
+            self.buttons.override_ind = True
+        
+        # MicroSDToastView blocks any attempts to use the Renderer in another thread so it
+        # never gives up the lock until it returns.
+        with self.renderer.lock:
+            if action == "remove":
+
+                toast = ToastOverlay(
+                    icon_name=FontAwesomeIconConstants.SDCARD,
+                    color=GUIConstants.NOTIFICATION_COLOR,
+                    label_text="MicroSD removed"
+                )
+                toast.render()
+                self.renderer.show_image()
```

Comment:
Shouldn't be up to a `View` to poke the `renderer`. When we command the `ToastOverlay` to do its thing, it should render and paint its pixels itself.